cli/gluster-block.c

```diff
@@ -1037,7 +1037,7 @@ main(int argc, char *argv[])
     goto fail;
   }
 
-  if(initLogging()) {
+  if(initLogging("cli")) {
```

Comment:
Can we use:
gbProcessCtx gbCtx = GB_CLI_MODE; /* set process mode */   ?

Reply:
WIll fix it.
Thanks.